File: src(java)/env.java
import java.nio.file.*;
import java.util.*;

public class env {
    private static final Map<String, String> vars = new HashMap<>();

    public static void load() {
        try {
            List<String> lines = Files.readAllLines(Path.of(".env"));
            for (String line : lines) {
                if (line.contains("=")) {
                    String[] parts = line.split("=", 2);
                    vars.put(parts[0].trim(), parts[1].trim());
                }
            }
        } catch (Exception e) {
        }
    }

    public static String get(String key) {
        return vars.getOrDefault(key, System.getenv(key));
    }
}

File: src/env.py
# env.py

variables = {}

def load(path=".env"):
    for line in open(path, "r"):
        line = line.strip()
        if not line or line.startswith("#"):
            continue
        key, value = line.split("=", 1)
        variables[key] = value

def get(key):
    return variables.get(key)
